python/mxnet/optimizer/optimizer.py

```diff
@@ -63,8 +63,10 @@ class Optimizer(object):
     clip_gradient : float, optional, default None
         Clip the gradient by projecting onto the box ``[-clip_gradient, clip_gradient]``.
 
-    learning_rate : float, optional, default 0.01
-        The initial learning rate.
+    learning_rate : float, optional, default None
+        The initial learning rate. If None, the optimization will use the
+        learning rate from ``lr_scheduler``. If not None, it will overwrite
```

Comment:
If None and lr_scheduler is None, lr is 0.01 by default.

Reply:
It is technically `None` by default, but effectively it is `0.01`. I'll include it in the params' doc.